release/_publishing.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 function add_fixed_issues_to_patcher_project_version {
-	IFS=',' read -r -a fixed_issues_array < "${_BUILD_DIR}/release/release-notes.txt"
+	lc_download "https://releases.liferay.com/dxp/${_PRODUCT_VERSION}/release-notes.txt" release-notes.txt
```

Comment:
@kiwm, can you please check if the file was indeed downloaded?

If not then we should display an error message and use the code `LIFERAY_COMMON_EXIT_CODE_BAD`